### PR TITLE
Add support and docs for sphinx-autobuild usage

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: presto-docs/README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ Sphinx and the included `make` script.
 Sphinx installation instructions for various operating systems and packaging
 systems are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
 
+The `pip`-based installation is preferred by the Presto team.
```
Comment (kokosing) [inline presto-docs/README.md]: I think this does not belong to this commit. Also I don't think this is something that should be mentioned and I was not aware of such preference.

File: presto-docs/README.md [hunk 2]
```diff
@@ -98,6 +100,22 @@ python3 -m http.server 4000
...
+changes. This is the fastest and best way to work on the documentation.
+
+To use it, simply install sphinx-autobuild, and then run
+
+```bash
+make clean livehtml
+```
+
+From now on the docs are available at
+[http://localhost:8000](http://localhost:8000).
```
Comment (kokosing) [inline presto-docs/README.md]: Nice!

File: presto-docs/README.md (hunk 1 above)
Comment (mosabua) [reply presto-docs/README.md]: I added it since sphinx-autobuild is best installed with pip .. I will move it down.

Comment (mosabua) [reply presto-docs/README.md]: done

**Comment by kokosing** [comment]
Merged, thanks!
